doc/handling-high-traffic.md

```diff
@@ -0,0 +1,29 @@
+# Handling High Traffic
+
+## Introduction
+Starting in 2025 Wikibase cloud started occasionally receiving a large amount of traffic.
+
+This has caused a number of incidents:
+- https://wmde.github.io/wikibase-cloud-status/issues/2025-02-10-nginx-oom/ / T385969
+- https://wmde.github.io/wikibase-cloud-status/issues/2025-07-15-scrapers-again/ / T399439 and T400046
+
+It's wasted quite a lot of engineering time; if you come here after dealing with this in the future why not update below.
+
+Wasted Engineering days: 26
+
+## Solutions in place
```

Comment:
We have also set the `$wgMaxExecutionTimeForExpensiveQueries` MediaWiki setting, which limits the amount of time MediaWiki can execute a database query for.

Reply:
good point I forgot to add this; done!